src/server/scripts/Outland/TempestKeep/Mechanar/boss_mechano_lord_capacitus.cpp

```diff
@@ -134,9 +139,19 @@ class boss_mechano_lord_capacitus : public CreatureScript
                             break;
                         case EVENT_SUMMON_NETHER_CHARGE:
                         {
-                            Position pos = me->GetRandomNearPosition(5.0f);
-                            me->SummonCreature(NPC_NETHER_CHARGE, pos, TEMPSUMMON_TIMED_DESPAWN, 18000);
-                            events.ScheduleEvent(EVENT_SUMMON_NETHER_CHARGE, 10s);
+                            uint32 spellId;
+                            uint32 netherChargeTimer;
```

Comment:
you can initialize the variable right away:
uint32 netherChargeTimer = DUNGEON_MODE(urand(9000, 11000), urand(2000, 5000));

uint32 spellId should also get initialized with 0 at least. It's not necessary because it's covered below but just to keep standards.